recipe/meta.yaml

```diff
@@ -1,18 +1,13 @@
-{% set version = "5.0.0" %}
+{% set version = "5.1.0" %}
 
 package:
   name: pyface
   version: {{ version }}
 
 source:
   fn: pyface-{{ version }}.tar.bz2
-  url: https://pypi.python.org/packages/source/p/pyface/pyface-{{ version }}.tar.bz2
-  md5: 0468d6a44adba0b70ec5171d1af9350a
-  patches:
-    # fix for issue https://github.com/enthought/pyface#188 following the
-    # solution from https://github.com/enthought/pyface/commit/06d85f8
-    # and raise syntax fix from https://github.com/enthought/pyface/pull/211
-    - pyface_fix.patch
+  url: https://pypi.python.org/packages/00/ec/04b1d7f1981107cde01bbc4a53ae2a234493e694cfa880fc00817e6c2a42/pyface-{{ version }}.tar.bz2
```

Comment:
I prefer PyPI. My thought is this. Someone can change GitHub tags after the fact. So, there is no guarantee on what we are getting. PyPI OTOH does make it near to (if not completely) impossible to change a release after it is made. That really let's us control what we work with. Further it indicates that developer(s) have committed to this being the release.